frontend/src/components/navigation/SideBar.vue

```diff
@@ -11,48 +11,21 @@
     :width="$vuetify.breakpoint.xl || (!mini && !$vuetify.breakpoint.mdAndUp) ? 350 : 256"
     :color="!title || mini ? 'blue-grey lighten-4' : null"
   >
-    <template v-if="title">
-      <v-list class="py-0">
-        <v-list-item v-if="mini" class="py-1" @click.stop="overrideExpanded = true">
-          <v-icon>{{ icon }}</v-icon>
-        </v-list-item>
-        <v-list-item v-else class="py-1 pr-2">
-          <v-list-item-title class="subtitle-1 font-weight-bold">
-            {{ title }}
-          </v-list-item-title>
-          <v-btn icon @click.stop="overrideExpanded = false">
-            <v-icon>mdi-chevron-left</v-icon>
-          </v-btn>
-        </v-list-item>
-      </v-list>
-      <v-divider />
-      <slot v-if="!mini" />
-    </template>
-    <template v-else>
```

Comment:
we don't need this because all usages now use the title?
Maybe you can mark the title property then as required?